BUILD

```diff
@@ -290,7 +290,7 @@ docker_container_image(
     operating_system = "linux",
     architecture = "amd64",
     base = "@typedb-ubuntu-x86_64//image",
-    cmd = ["/opt/typedb-server-linux-x86_64/typedb", "server"],
+    cmd = ["/opt/typedb-server-linux-x86_64/typedb", "server", "--storage.data-directory=/var/lib/typedb/data"],
```

Comment:
Can you check if we can pass arguments to typedb? I think we may have to use 'entrypoint' as well:
```
entrypoint = ["/opt/typedb-server-linux-x86_64/typedb", "server"],
cmd = ["--storage.data-directory=/var/lib/typedb/data"],
```